<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>output2</class>
 <widget class="QWidget" name="output2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>413</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>372</width>
    <height>236</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cведения о непериодических изданиях, вышедших тиражом не менее указанного</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Тираж:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Вывести</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Закрыть</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Наименование</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Год</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Периодичность</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тираж</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Издательство</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>output2</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
